src/customers/customer-sign-in.md

```diff
@@ -81,3 +81,14 @@ _Sign In_
 
    ![]({% link images/images-b2b/customer-company-dashboard.png %}){: .zoom}
    _Account Dashboard - company_
+
+## Sign out of Customer Account
+
+1. In the upper-right corner, click the **Welcome, Customer Name!** <span class="btn">∨</span> icon.
```

Comment:
@ZhovnirMaria could you please use `top right` instead of `upper-right`. 

Thank you!

Reply:
@atwixfirster  
Done. Thank you!